### ISSUE TITLE
How to get loss value per epoch?

### ISSUE THREAD

**Issue by robobachu** [thieu1995/mealpy#58]
For the latest version (2.4.0) how to get the best loss value per epoch?

**Comment by thieu1995** [comment]
Hi @robobachu 

You can get those values from class History. Document here (https://mealpy.readthedocs.io/en/latest/pages/models/mealpy.utils.html#module-mealpy.utils.history)

```
print(model.history.list_global_best_fit)
print(model.history.list_current_best_fit)
```

**Comment by robobachu** [comment]
I tried, but when printing the model.history.list_global_best_fit value, I not only see the loss values per epoch, but also the SOLUTION unlike the previous version, I think it should be fixed like the session the previous version makes more sense because there is history.list_global_best which stores SOLUTION information by epoch

**Comment by thieu1995** [comment]
@robobachu ,

Thank you for suggesting. I've checked the code and found some issues related to this variable. I will fix it in the next version. In the meantime, you can get the loss values by this simple function.

```
def get_loss(list_global_best):
	loss = []
	for agent in list_global_best:
		loss.append(agent[1][0])
	return loss 

get_loss(model.history.list_global_best)
```

**Comment by robobachu** [comment]
Thanks for your support

**Comment by thieu1995** [comment]
@robobachu ,

oh, I forgot to use comprehension list in python. You actually don't need the function above. You just need this line of code to get loss. 

`loss = [agent[1][0] for agent in model.history.list_global_best] `
